src/styles/calendar.scss

```
@@ -0,0 +1,45 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: $cell-width * 7 + $gap * 6;
+  padding: 30px $set-padding 0;
+  gap: $gap;
+
+  @each $day, $number in $days {
+    &--start-day--#{$day} :nth-child(-n + 1) {
+      margin-left: calc(
+        (#{$gap } * (#{$number} - 1)) + (#{$cell-width} * (#{$number} - 1))
```

```suggestion
        ($gap * ($number - 1) + ($cell-width * ($number - 1))
```
You can use variables as follow:

---

Thanx!